worlds/aquaria/Regions.py

```diff
@@ -0,0 +1,1377 @@
+            state.has("Octopus Prime beated", player) and
+            state.has("Crabbius Maximus beated", player) and
+            state.has("Mantis Shrimp Prime beated", player) and
+            state.has("King Jellyfish God Prime beated", player)
+            )
+
+def _has_secrets(state, player: int) -> bool:
+    return (state.has("First secret obtained",player) and
+            state.has("Second secret obtained",player) and
+            state.has("Third secret obtained",player))
+
+
+class AquariaRegions:
+    """
+    Class used to create regions of the Aquaria game
+    """
+    menu: Region
+    verse_cave_r: Region
+    verse_cave_l: Region
+    home_water: Region
+    home_water_nautilus: Region
+    home_water_transturtle: Region
+    naija_home: Region
+    song_cave: Region
+    energy_temple_1: Region
+    energy_temple_2: Region
+    energy_temple_3: Region
+    energy_temple_boss: Region
+    energy_temple_idol: Region
+    energy_temple_blaster_room: Region
+    energy_temple_altar: Region
+    openwater_tl: Region
+    openwater_tr: Region
+    openwater_tr_turtle: Region
+    openwater_bl: Region
+    openwater_br: Region
+    skeleton_path: Region
+    skeleton_path_sc: Region
+    arnassi: Region
+    arnassi_path: Region
+    arnassi_crab_boss: Region
+    simon: Region
+    mithalas_city: Region
+    mithalas_city_top_path: Region
+    mithalas_city_fishpass: Region
+    cathedral_l: Region
+    cathedral_l_tube: Region
+    cathedral_l_sc: Region
+    cathedral_r: Region
+    cathedral_underground: Region
+    cathedral_boss_l: Region
+    cathedral_boss_r: Region
+    forest_tl: Region
+    forest_tl_fp: Region
+    forest_tr: Region
+    forest_tr_fp: Region
+    forest_bl: Region
+    forest_br: Region
+    forest_boss: Region
+    forest_boss_entrance: Region
+    forest_sprite_cave: Region
+    forest_sprite_cave_tube: Region
+    mermog_cave: Region
+    mermog_boss: Region
+    forest_fish_cave: Region
+    veil_tl: Region
+    veil_tl_fp: Region
+    veil_tr_l: Region
+    veil_tr_r: Region
+    veil_bl: Region
+    veil_b_sc: Region
+    veil_bl_fp: Region
+    veil_br: Region
+    octo_cave_t: Region
+    octo_cave_b: Region
+    turtle_cave: Region
+    turtle_cave_bubble: Region
+    sun_temple_l: Region
+    sun_temple_r: Region
+    sun_temple_boss_path: Region
+    sun_temple_boss: Region
+    abyss_l: Region
+    abyss_lb: Region
+    abyss_r: Region
+    ice_cave: Region
+    bubble_cave: Region
+    king_jellyfish_cave: Region
+    whale: Region
+    first_secret: Region
+    sunken_city_l: Region
+    sunken_city_r: Region
+    sunken_city_boss: Region
+    sunken_city_l_bedroom: Region
+    body_c: Region
+    body_l: Region
+    body_rt: Region
+    body_rb: Region
+    body_b: Region
+    final_boss_loby: Region
+    final_boss_tube: Region
+    final_boss: Region
+    final_boss_end: Region
+    """
+    Every Region of the game
+    """
+
+    world: MultiWorld
```

Comment:
```suggestion
    multiworld: MultiWorld
```

To reduce confusion, especially when others look at your world for references